### ISSUE TITLE
Colours should only be shown when the output is a terminal

### ISSUE THREAD

**Issue by wilfriedvanasten** [aurapm/aura#102]
I tend to use grep/less and all kinds of other commands on pacman's search quite a lot. Now I wanted to do this on aura -As search to, but the output is just not that nice because of the escape sequences that would control coloring on terminals:

A sample of output piped into less:

ESC[0;35maur/ESC[0mESC[mESC[1mgtk2hs-ESC[0mESC[1;36mdarcsESC[0m ESC[0;32m20100205-1ESC[0m (ESC[0;33m0ESC[0m)
    A GTK+2 binding for Haskell

This can be avoided by only outputting this if the stdout handle is a terminal. I have to admit I have no idea how to do this in haskell, but I know it is possible. An alternative would be to not use escape sequences but a tty API to control coloring.

**Comment by v1kn** [comment]
I have just encountered this issue too. Unfortunately, while piping `-As` command through `less -R` produces a clean output, redirecting it to a file renders the output unintelligible. I compared this to other colored AUR helpers (`yaourt`, `burgaur`), they do not have this issue.  

Seeing `Aria2` tag, I understand that it will not be implemented in current branch. Is there a TBR date for `Aria2`? It is still the best AUR helper, and I'd love to see this issue fixed.  

Thank You
